contrib/scripts/isc_dhpcd_graphite.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""Send dhcpd statistics generated by dhcpd-pools to graphite
+
+Does not attempt to reconnect if the server is unavailable.
+"""
+
+import argparse
+from binascii import hexlify
+from collections import namedtuple
+import json
+from math import trunc
+import pathlib
+import pickle
+import re
+import socket
+import struct
+import subprocess
+import sys
+from time import time
+
+DEFAULT_PREFIX = "nav.dhcp"
+DEFAULT_CONFIG_FILE = "/etc/dhcpd/dhcpd.conf"
+DEFAULT_CMD_PATH = pathlib.Path("/usr/bin/dhcpd-pools")
+DEFAULT_PROTOCOL = 2  # Python 3: 3, Python 3.8+: 4
```

Comment:
The Graphite Pickle protocols are iffy. While it's nice that you included support for them, I would prefer if the `DEFAULT_PROTOCOL` was `'text'`, as that is compatible no matter what Python or Graphite versions you run.